i18n/checkLocaleKeys.js

```diff
@@ -21,6 +21,25 @@ const checkLocaleKeys = () => {
 
     const referenceKeys = extractAllKeys(enFileContent)
 
+    //check if in CI/CD
+    if (process.env.NODE_ENV === 'production') {
+        const missingKeysSummary = findAndReportMissingKeysInNonEnFiles(
```

Comment:
this could be a function to distinguish what's happning in this vs the normal flow.